readme.md

```diff
@@ -412,6 +415,10 @@ interface SubmitTerms {
 The extension MUST only send the request with active consent of the user.
 This is the first step where the user’s DID is revealed to the dApp.
 
+If the `owner` DID URI was provided and the extension controls this DID, then the response SHOULD include the same `owner`. 
+Legacy attesters provide meaningless values in this field, so if the extension does not control this DID,
+then the response MUST include `owner` set to the DID the user chooses.
```

Comment:
I find this description still rather vague

```suggestion
If the credential proposal contains an `owner` field indicating for which DID the attester is willing to issue a credential,
extensions SHOULD request the user's permission for requesting a credential to be issued to that DID.
Note that legacy attesters provide meaningless values in this field, so if the extension does not control this DID,
extensions MUST ask their users to select a DID for which the credential shall be issued.

Attester application MUST only issue a credential to the DID in contained in the claimer's request.
Attesters MAY choose to reject any request indicating a different DID to be used than the one for which they offered a credential proposal.
```

Reply:
I’ve expanded your suggestions. How do you like it now?